Applications/Spire/Include/Spire/Spire/ShuttleQtTypes.hpp

```diff
@@ -78,6 +78,9 @@ namespace Beam {
       sender.send("point_size", value.pointSize());
       sender.send("weight", value.weight());
       sender.send("italic", value.italic());
+      if(value.pointSize() == -1) {
```

Comment:
Don't conditionally save values. Save the point size no matter what. Then later on load the point size back in and you can ignore the point size if it's ```-1```.